js/state/syncUtil.js

```diff
@@ -39,18 +39,43 @@ const siteSettingDefaults = {
   ledgerPaymentsShown: true
 }
 
+// Shared between bookmarks and history sites.
+const SITE_FIELDS = ['objectId', 'location', 'title', 'customTitle', 'tags', 'favicon', 'themeColor', 'lastAccessedTime', 'creationTime', 'partitionNumber', 'parentFolderId']
```

Comment:
[minor] partitionNumber is only useful on laptop since AFAIK the other clients don't have partitions